libs/model/src/models/reaction.ts

```diff
@@ -81,6 +119,7 @@ export default (
                         event_payload: {
                           ...reaction.get({ plain: true }),
                           reaction: 'like',
+                          contestManagers,
```

Comment:
do we need all contest manager fields here? I think identifiers are enough

Reply:
removed all except contest address